lib/Models/Terria.ts

```diff
@@ -620,14 +635,18 @@ export default class Terria {
         typeof document !== "undefined" ? document.baseURI : "/"
       ).toString();
     }
+
     if (options.baseUrl) {
-      if (options.baseUrl.lastIndexOf("/") !== options.baseUrl.length - 1) {
-        this.baseUrl = options.baseUrl + "/";
-      } else {
-        this.baseUrl = options.baseUrl;
-      }
+      this.baseUrl = ensurePrefix(options.baseUrl, "/");
     }
 
+    this.cesiumBaseUrl = ensurePrefix(
+      options.cesiumBaseUrl ?? `${this.baseUrl}build/Cesium/build`,
```

Comment:
```suggestion
      options.cesiumBaseUrl ?? `${this.baseUrl}build/Cesium/build/`,
```

Provide the default in the preferred form.